Dockerfile

```diff
@@ -1,56 +1,56 @@
 # Build topolvm
-FROM --platform=$BUILDPLATFORM golang:1.20-bullseye AS build-topolvm
+FROM --platform=$BUILDPLATFORM golang:1.20.13-alpine3.19 AS build-topolvm
 
 # Get argument
 ARG TOPOLVM_VERSION
 ARG TARGETARCH
 
+RUN apk add --no-cache \
+    make bash
+
 COPY . /workdir
 WORKDIR /workdir
 
 RUN touch lvmd/proto/*.go
 RUN make build-topolvm TOPOLVM_VERSION=${TOPOLVM_VERSION} GOARCH=${TARGETARCH}
 
-# TopoLVM container
-FROM --platform=$TARGETPLATFORM ubuntu:18.04 as topolvm
-
-ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update \
-    && apt-get -y install --no-install-recommends \
-        btrfs-progs \
-        file \
-        xfsprogs \
-    && rm -rf /var/lib/apt/lists/*
-
-COPY --from=build-topolvm /workdir/build/hypertopolvm /hypertopolvm
-
-RUN ln -s hypertopolvm /lvmd \
-    && ln -s hypertopolvm /topolvm-scheduler \
-    && ln -s hypertopolvm /topolvm-node \
-    && ln -s hypertopolvm /topolvm-controller
-
-COPY --from=build-topolvm /workdir/LICENSE /LICENSE
-
-ENTRYPOINT ["/hypertopolvm"]
-
 # Build sidecars
 FROM --platform=$BUILDPLATFORM build-topolvm as build-sidecars
 
 # Get argument
 ARG TARGETARCH
 
-ENV DEBIAN_FRONTEND=noninteractive
-RUN  apt-get update \
-    && apt-get -y install --no-install-recommends \
-        patch
+RUN apk add --no-cache \
+    make patch curl
 
 RUN make csi-sidecars GOARCH=${TARGETARCH}
 
 # TopoLVM container with sidecar
-FROM --platform=$TARGETPLATFORM topolvm as topolvm-with-sidecar
+FROM --platform=$TARGETPLATFORM alpine:3.19 as topolvm-with-sidecar
+
+RUN apk add --no-cache \
```

Comment:
This will add licensing issues to the released TopoLVM images as they now package various OS packages that need to be mentioned in the licensing

Reply:
Honestly this is a great point. @toshipp I suggest we open up an issue on LICENSE tracking for all of our open source dependencies in TopoLVM so that we remain compliant in the project and questions like these dont arise in PRs. I suggest we do this before we merge here.

We (speaking on behalf of Red Hat) are interested in remaining License Compliant for TopoLVM and we personally do not use the Dockerfile in the open source version because we rebuild with Red Hat Maintained versions of packages such as lvm2 (which is also maintained and owned by Red Hat and published as open source).

Until this is not cleared up however, I suggest not to move forward with this change to make sure no licensing aspects change while we move for more transparency.

Reply:
I also see the way this is being built today the sidecars from the kube project are embeded into the container, I don't see a THIRDPARTY or other sbom so I would like to continue with the change as its more likely slightly better due to alpine being a smaller base image with fewer inclusions, or a net same

Reply:
Its a different story if you use third party container images or if you build your own container image imo. But lets wait for other maintainers opinions

Reply:
https://github.com/topolvm/topolvm/issues/818 for LICENSE discussions. Thanks for creating!

Reply:
@jakobmoellerdev @ryanfaircloth I'll handle #818.